### PR TITLE
[iOS] Distribute Beta iOS App with TestFlight external testers with XcodeCloud

### PR DESCRIPTION
**Pull request by ry-itto**
## Issue
- close #ISSUE_NUMBER
- [x] Update iOS README about public link (TestFlight external test)
- remove android reference in strings, assets

## Overview (Required)
-

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by ry-itto** [comment]
@takahirom
Please check Kotlin/Android part 🙏🏼

**Comment by takahirom** [comment]
👀

File: app-ios/README.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ cd ../
  open DroidKaigi2022/DroidKaigi2022.xcodeproj
 ```
 
+## Try it out
```
Comment (takahirom) [inline app-ios/README.md:13]: I think you can add it in README.

## Try it out
### Android
...
### iOS
....

File: app-ios/Sources/Strings/Resources/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -3,7 +3,7 @@
 // About
 "About.Title" = "DroidKaigiとは";
 "About.WhatIsDroidKaigi" = "What is DroidKaigi?";
-"About.Description" = "DroidKaigiはエンジニアが主役のAndroidカンファレンスです。\nAndroid技術情報の共有とコミュニケーションを目的に、2022年10月5日(水)〜7日(金)の3日間開催します。";
+"About.Description" = "DroidKaigiはエンジニアが主役のカンファレンスです。\n技術情報の共有とコミュニケーションを目的に、2022年10月5日(水)〜7日(金)の3日間開催します。";
```
Comment (takahirom) [inline app-ios/Sources/Strings/Resources/en.lproj/Localizable.strings:6]: Now that we merge https://github.com/DroidKaigi/conference-app-2022/pull/387, I think we can use them. But it is OK now. 🙏

Comment (ry-itto) [reply app-ios/Sources/Strings/Resources/en.lproj/Localizable.strings:6]: I'll check that after this PR merged 🙏🏼

**Review by takahirom — APPROVED** [review]
Looks great!

**Pull request merged**